emails/payin_directdebit_succeeded.spt

```diff
@@ -1,8 +1,7 @@
-{{ _("The direct debit payment to your Liberapay wallet has succeeded") }}
+{{ _("The direct debit payment to your Liberapay wallet succeeded") }}
```

Comment:
succeeded is in the past, so no need for "has".

Reply:
Same as above